Framework/basic.json

```diff
@@ -39,7 +39,11 @@
         "className": "o2::quality_control_modules::skeleton::SkeletonTask",
         "moduleName": "QcSkeleton",
         "detectorName": "TST",
-        "cycleDurationSeconds": "10",     "": "10 seconds minimum",
+        "": "The last item in cycleDurations will be used for the rest of the duration whatever the period",
+        "cycleDurations": [
+          {"cycleDuration": 10, "period": 35},
+          {"cycleDuration": 12, "period": 1}
```

Comment:
as in O2, can you use `validity` instead of `period`? You could also consider adding back `seconds` somewhere.

Reply:
yes